tests/test_orientations.py

```diff
@@ -53,7 +53,8 @@ def test_orientations_from_view_up_invalid():
     views = [[1, 0, 0], [0, 0]]
     ups = [[0, 1, 0], [0, 0, 0]]
     with raises(ValueError):
-        Orientations.from_view_up(views, ups)
+        with warns(np.VisibleDeprecationWarning):
+            Orientations.from_view_up(views, ups)
```

Comment:
What was this warning - it sounds like there might be action required on our side.

Reply:
It says "VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray."
It refers to calling `np.asarray(views)` with `views = [[1, 0, 0], [0, 0]]`, which in the end also causes the ValueError. So, I do think there is action required from our side.